app/client/components/helpCentre/helpCentreFaq.tsx

```diff
@@ -0,0 +1,159 @@
+import { css } from "@emotion/core";
+import { space } from "@guardian/src-foundations";
+import { neutral } from "@guardian/src-foundations/palette";
+import { textSans } from "@guardian/src-foundations/typography";
+import React, { useState } from "react";
+import { trackEvent } from "../analytics";
+import { helpCentreMoreQuestionsConfig } from "./helpCentreConfig";
+import { linkAnchorStyle, linkArrowStyle } from "./HelpTopicBox";
+
+const faqStyles = css({
+  marginBottom: "10px",
+  display: "flex",
+  flexWrap: "wrap",
+  textAlign: "left",
+  fontWeight: "normal"
+});
+
+export const HelpCentreFaq = () => {
+  const [indexOfOpenSection, setIndexOfOpenSection] = useState<number>();
+
+  const containterCss = `
+    width: 100%;
+    border: 1px solid ${neutral["86"]};
+  `;
+
+  const sectionTitleCss = (isOpen: boolean, isNotFirstOption: boolean) => `
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: ${neutral["7"]};
+    ${textSans.medium()};
+    margin: 0;
+    padding: ${space[3]}px ${space[3] * 2 + 15}px ${space[3]}px ${space[3]}px;
+    position: relative;
+    cursor: pointer;
+    :after {
+      content: "";
+      display: block;
+      width: 7px;
+      height: 7px;
+      border-top: 2px solid ${neutral["7"]};
+      border-right: 2px solid ${neutral["7"]};
+      position: absolute;
+      top: 50%;
+      transform: translateY(-50%) ${
+        isOpen ? "rotate(-45deg)" : "rotate(135deg)"
+      };
+      transition: transform 0.4s;
+      right: 17px;
+    }
+    ${isNotFirstOption &&
+      `
+      :before {
+        content: "";
+        display: block;
+        position: absolute;
+        top: 0;
+        left: 0px;
+        width: 100%;
+        height: 1px;
+        background-color: ${neutral["86"]}
+      }
+    `}
+  `;
+
+  const innerSectionCss = (isOpen: boolean) => `
```

Comment:
Is there a reason to declare these as functions instead of dynamic styles like we do elsewhere?

Reply:
Is there an issue with declaring these as functions?

Reply:
I was wondering if there's a benefit to this, as it's a change of pattern from what we do in project

Reply:
Ah right, there are few examples of in the codebase using this method at the mo

Reply:
We have inline styled components, yes. But not functions declared and then converted inline into dynamic styles.

Reply:
Have a search in the codebase, you should see a few examples